.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please include a summary of the change and which (if so) issue is fixed.
+
+Fixes #(ISSUE_NUMBER_HERE)
+
+## Checklist
+
+Please delete options that are not relevant.
+
+- [ ] Fixed style issues using `pre-commit run --all` (first install using `pip install pre-commit`)
+- [ ] `pytest` passed
+
+## Type of change
+
+*Please delete options that are not relevant.*
```

Comment:
Having a list of checkboxes *and* saying to delete irrelevant options is redundant.

```suggestion
*Check relevant option(s).*
```

Reply:
Also fixed 👍

Reply:
Sorry, I didn't see the commit suggestion in time. I had already amended and force pushed.